.eslintrc.cjs

```diff
@@ -11,6 +11,7 @@ module.exports = {
     '@electron-toolkit/eslint-config-prettier',
   ],
   rules: {
+    '@typescript-eslint/no-explicit-any': 'off',
     '@typescript-eslint/no-unused-vars': 'off',
```

Comment:
I just reverted this change, since there are many `/* eslint-disable @typescript-eslint/no-explicit-any */` comments, maybe we need to confirm whether we disable it globally and remove all of the comments later.